projects/packages/backup/src/class-rest-controller.php

```diff
@@ -688,6 +699,30 @@ public static function fetch_wc_orders_backup( $request ) {
 		);
 	}
 
+	/**
+	 * Fetch backup preflight status
+	 *
+	 * @return array
+	 */
+	public static function get_site_backup_preflight() {
+		$blog_id = Jetpack_Options::get_option( 'id' );
+
+		$response = Client::wpcom_json_api_request_as_user(
+			'/sites/' . $blog_id . '/rewind/preflight?force=wpcom',
+			'v2',
+			array(),
+			null,
+			'wpcom'
+		);
+
+		if ( 200 !== wp_remote_retrieve_response_code( $response ) ) {
+			return null;
+		}
```

Comment:
It would be better if here you created and returned a `WP_Error` instance with the error message that came from `$response`, as that way we might know what exactly went wrong.

Reply:
You are right! I'll add it.

Reply:
@pypt done! Added more details when there is an error :)